managed-os/amilookup.js

```diff
@@ -35,7 +41,10 @@ exports.handler = function(event, context) {
 
     if (firstword == "Windows") {
       owner = "amazon";
-    } else {
+    } else if (firstword == "CIS") {
+        owner = "679593333241";
+    }

```

Comment:
Owner will be aws-marketplace for CIS image

Reply:
@bhargavapratik  Account ID for "aws-marketplace" is "679593333241". Both are same since account id was hardened for linux os type thats why i hardened the account Id for CIS image type.

Reply:
@bhargavapratik  addressed